src/inmanta/execute/proxy.py

```diff
@@ -18,9 +18,10 @@
 
 from collections.abc import Iterable, Mapping, Sequence
 from copy import copy
-from typing import Callable, Optional, Union
+from typing import Callable, Union
 
-from inmanta.ast import NotFoundException, RuntimeException
+# Keep UnsetException, UnknownException and AttributeNotFound in place for backward compat with <iso8
+from inmanta.ast import AttributeNotFound, NotFoundException, RuntimeException, UnknownException
```

Comment:
Probably need a noqa in here as well.

Reply:
they are actually used, if it passed, it is fine